index.html

```diff
@@ -197,5 +197,33 @@ <h3>Skills</h3> <img src="Images/Arrow_right.svg" alt="Arrow_right">
       
   </section>
 
+  <section class="all-contact">
+
+    <div class="contact-background">
+
+    <div class="margins">
+      
+      <h2>Contact me</h2>
+      
+      <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p>
+      
+      <form class="boxels" action="https://formspree.io/f/myyvzedv" method="post">
+
+        <input class="contact-boxes" type="text" id="name" name="user_name" maxlength="30" placeholder="your name here" required>
+        
+        <input  class="contact-boxes" type="email" id="mail" name="user_email" placeholder="your email address" required>
+        
+        <textarea  class="contact-boxes" cols="30" rows="6" id="contact-msg" name="user_message" maxlength="500" placeholder="Write your message here" required></textarea>
+       
+        <div class="project-check"> <button type="submit">Get it touch</button></div>
```

Comment:
- [ ]  Kindly make sure that Button interactions such as hover, active (when it's pressed), disabled, and enable (default state) are implemented as shown in the Figma template. please fix this as it was requested by previous reviewer 👍🏿 
- Check the screenshot below:


 ![image](https://user-images.githubusercontent.com/71874425/186142037-41a9e629-218f-4ab5-9f54-cf90ae740945.png)